src/lib/loaders/__tests__/batchLoader.test.ts

```diff
@@ -79,55 +79,45 @@ describe("batchLoader", () => {
 
       expect(multipleLoader).toBeCalledTimes(1)
     })
-
-    it("should return a default value when an id is missing", async () => {
-      const multipleLoader = jest.fn(({ id: ids }) =>
-        ids.slice(1).map(id => ({
-          _id: id,
-        }))
-      )
-      const batch = batchLoader({ multipleLoader, defaultResult: {} })
-
-      const results = await Promise.all([
-        batch("123"),
-        batch("456"),
-        batch("789"),
-      ])
-      expect(results).toEqual([
-        {},
-        {
-          _id: "456",
-        },
-        {
-          _id: "789",
-        },
-      ])
-
-      expect(multipleLoader).toBeCalledWith({
-        id: ["123", "456", "789"],
-        size: 3,
-      })
-    })
   })
 })
 
-const { groupKeys } = require("../batchLoader")
-describe("groupKeys", () => {
+describe("groupByParams", () => {
+  const { groupByParams } = require("../batchLoader")
   it("should handle a single key", () => {
-    expect(groupKeys(["a"])).toEqual([
-      {
-        id: ["a"],
-        size: 1,
-      },
+    expect(groupByParams([{ id: "a" }])).toEqual([
+      [""],
```

Comment:
This value here makes understanding the behaviour a little harder, it could be improved by first having a test that just focusses on this value and thus explains why this is blank.

Reply:
I see how that can be confusing. I added tests for `serializeParams` right about this that determines what that behavior would be.